gce-production-1/main.tf

```diff
@@ -63,8 +63,8 @@ module "gce_project_1" {
   worker_account_json_com       = "${file("${path.module}/config/gce-workers-production-1.json")}"
   worker_account_json_org       = "${file("${path.module}/config/gce-workers-production-1.json")}"
   worker_image                  = "${var.gce_worker_image}"
-  worker_instance_count_com     = 36
-  worker_instance_count_org     = 40
+  worker_instance_count_com     = 16
```

Comment:
Is this change intentionally included here? (just asking because it wasn't mentioned in the description)

Reply:
Ah, thank you for bringing this up!  There's a bit in the commit message, but I forgot to update the PR description.  I made this change to reflect more closely the counts of hosts that were running in gce-production-1 at the time.  The majority of gce-production-1 hosts were in a `TERMINATED` state due to actions that I took in late 2017 prior to holiday time code/config freeze.